models/public/erfnet/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This is a ONNX* version of `ERFNet` model designed to perform real-time lane detection.
+  This model is pre-trained in PyTorch* framework and retrained by CULane.
+  For details see repository <https://github.com/Zhangxianwen2021/ERFNet>,
+  paper of ERFNet<https://doi.org/10.1109/TITS.2017.2750080>and repository <https://github.com/cardwing/Codes-for-Lane-Detection/tree/master/ERFNet-CULane-PyTorch>
+task_type: instance_segmentation
+files:
+  - name: erfnet.onnx
+    size: 8253755 
+    checksum: f6597d416f02b8218b557c4f673a0837fc215213055f9957267667003578e76507515d7c2be5d6983549f81cb96a0a66
+    source: https://github.com/Zhangxianwen2021/ERFNet/raw/main/erfnet.onnx
+model_optimizer_args:
+  - --input=input
+  - --input_shape=[1, 3, 208, 976]
```

Comment:
looks like you use out dated config format, now input should be specified as separated  section `input_info`, please follow
https://github.com/openvinotoolkit/open_model_zoo/blob/master/CONTRIBUTING.md